#Google - Python Exception
try :
    # result = 45/0
    result1 = 45/5
except:
    print('Error Happened')
finally:
    print('Finall Here')
print('Done') 